website/src/doc-demos/codepen.js

```diff
@@ -0,0 +1,219 @@
+
+const PROP_BLACK_LIST = ['dataComparator', 'fetch'];
+const BASE_LAYER_PROP_WHITE_LIST = ['autoHighlight', 'coordinateOrigin', 'coordinateSystem', 'highlightColor', 'modelMatrix', 'opacity', 'pickable', 'visible', 'wrapLongitude'];
+const PROP_OVERRIDES = {
+  coordinateSystem: 'COORDINATE_SYSTEM.LNGLAT',
+  renderSubLayers: 'props => new GeoJsonLayer(props)',
+  characterSet: '" !\\"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~"'
+};
+
+function addIndent(text, spaces) {
```

Comment:
Wow, that is a lot of dense processing code...  Automation is nice though. 

Add a JSDoc header on each function at least to make it bit less inscrutable?

Better filename `codepen-automation.js` or something?